docs/run-testing-framework.md

```diff
@@ -35,8 +35,16 @@ In the case that you want to debug for a certain platform, you can also use this
  
 #### 2.1.1 Setup your aws credentials
 Refer to: https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-files.html
+
+#### 2.1.2 Setup unique bucket ID
+
+First create a unique S3 bucket identifier. This needs to be globally unique. Change example to an UUID of your choice
```

Comment:
Should the docs here make clear that this is appended to a fixed portion of a bucket name and doesn't need to be a whole bucket name?  Maybe provide an example that generates a random value rather than just `example`?

```shell
export TF_VAR_bucketId=$(dd if=/dev/urandom bs=1k count=1k | shasum | cut -b 1-8)
```

Reply:
Yea this makes more sense.
Would changing the variable name away from `bucketId` help also? What could we call it instead? `bucketUUID`?